spec/models/audits/question_spec.rb

```diff
@@ -1,6 +1,6 @@
-RSpec.describe Question do
+RSpec.describe Audits::Question do
```

Comment:
If you enclose the test within `module Audits` there is no need to scope every class inside